R/loadInterp.R

```diff
@@ -315,7 +315,13 @@ predictSolute.loadInterp <- function(
     preds <- data.frame(date=getCol(load.model@metadata, newdata, "date"), preds)
   }
   
-  preds
+  if(lin.or.log == "log"){
```

Comment:
i don't think we're always returning a data.frame with columns for fit, se.fit, and se.preds - specifically, se.fit and se.preds and lwr and upr are all optional columns selected (or not) by the arguments. right? so there might need to be more conditionals in here.